test/e2e/specs/editor/plugins/child-blocks.spec.js

```diff
@@ -63,11 +61,9 @@ test.describe( 'Child Blocks', () => {
 			name: 'test/child-blocks-restricted-parent',
 		} );
 
-		await page
-			.getByRole( 'document', {
-				name: 'Block: Child Blocks Restricted Parent',
-			} )
-			.click();
+		await page.click(
+			'[data-type="test/child-blocks-restricted-parent"] .block-editor-default-block-appender'
+		);
```

Comment:
Same here.

Reply:
Hi @Mamaduka, I tried to use the solution here but it is not working as expected. 

`await page
				.getByRole( 'document', {
					name: 'Block: Child Blocks Restricted Parent',
				} )
				.getByRole( 'button', {
					name: 'Add block',
				} )
				.click();`

Reply:
I think this should work now:

```js
await page
			.getByRole( 'document', {
				name: 'Block: Child Blocks Restricted Parent',
			} )
			.getByRole( 'button', {
				name: 'Add default block',
			} )
			.click();
```